ocaml/seahash/libseahash/Makefile

```diff
@@ -0,0 +1,6 @@
+
+all:
+	gcc -O3 -std=c11 -c seahash.c -o libseahash_stubs.a
```

Comment:
This should use CC, that way you also get the hardening flags that an RPM build would usually set. You probably also get an easier way to run things like clang static analyzer and coverity if you allow CC to be overridable.